docs/getting-started/tutorials/auth-architecture/external-idp.md

```diff
@@ -0,0 +1,78 @@
+---
+id: external-idp 
+title: Integrate an external Identity Provider to your project
+sidebar_label: External Identity Provider
+---
+
+In order to provide a higher security level, you may need to protect your endpoints using an external Identity Provider (IDP) such as Okta, Keycloak, etc.
+
+## Definition
+
+![Scenario2](img/auth-scenario2.png)
+
+The involved microservice of the flow at runtime are:
+1. `API Gateway`: Mia-Platform plugin available in [Nginx](/runtime_suite/api-gateway/10_overview.md) or [Envoy](/runtime_suite/envoy-api-gateway/overview.md)
+2. [`Authorization Service`](/runtime_suite/authorization-service/10_overview.md): Mia-Platform plugin
+3. `Authentication Service`: a custom microservice that you need to implement. It must integrate with your external IDP to resolve the user token.
```

Comment:
Would be better to change the name to avoid confusions with our `Authentication Service`

Reply:
done